### PR TITLE
sci-libs/rocFFT fixes

### PR DESCRIPTION
**Pull request by littlewu2508**

### CODE REVIEW

File: sci-libs/rocFFT/rocFFT-4.3.0-r1.ebuild [hunk 1]
```diff
@@ -22,9 +22,9 @@ DEPEND="${RDEPEND}
 
 BDEPEND="
 	test? ( dev-cpp/gtest dev-libs/boost
-	>=sci-libs/fftw-3
+	>=sci-libs/fftw-3 sys-libs/libomp )
 	>=dev-util/cmake-3.22
-)"
+"
```
Comment (heroxbd) [inline sci-libs/rocFFT/rocFFT-4.3.0-r1.ebuild:27]: Explain in your comment message why this fixes https://bugs.gentoo.org/850937

**Review by heroxbd — CHANGES_REQUESTED** [review]

File: sci-libs/rocFFT/rocFFT-4.3.0-r1.ebuild (hunk 1 above)
Comment (littlewu2508) [reply sci-libs/rocFFT/rocFFT-4.3.0-r1.ebuild:27]: Done

**Comment by littlewu2508** [comment]
> You should give this ebuild a revision bump, because a lot of metadata (`BDEPEND`) are changed.

I wonder whether this can be treated as "adding a missing build-time dependency that caused a build failure" in https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html which does not need revision bump

**Comment by heroxbd** [comment]
littlewu2508 ***@***.***> writes:

> I wonder whether this can be treated as "adding a missing build-time
> dependency that caused a build failure" in
> https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html
> which does not need revision bump

Good. Your justification is sound.  We don't need revbump in any of the
3 commits.

**Pull request closed without merging**